fixed #214 removed height property from cards to make responsive

love-sonkar opened the pull request:
fixed #214 
i have fixed the responsive issues in cards.
Now cards content is not overflowing.

![image](https://github.com/user-attachments/assets/fb25ebcb-e67f-41e2-a4a9-aa57b975096a)

max-programming commented:
Thanks for submitting this fix! Would you be interested in addressing some other responsive issues I mentioned below?

![hacktoberfest-projects-git-for-fbdb9a-max-programmings-projects vercel app_repos_javascript(iPhone 12 Pro)](https://github.com/user-attachments/assets/1b5720f2-933e-435f-8db8-ed997dc1874e)

max-programming commented on package-lock.json:
Can you delete the package lockfile please?
As this project uses `pnpm`

love-sonkar replied:
sure

love-sonkar commented:
i have fixed the issue,
i'm sharing the image please review once
![image](https://github.com/user-attachments/assets/429b5cf2-c5d5-4ffe-b1de-8fbb5b70ab65)


i'm not able to reproduce this issue other than that i have fixed all issue as you mentions.

![Screenshot 2024-10-01 232931](https://github.com/user-attachments/assets/92a10a1c-6b00-4aa5-8b2e-21b368972352)

max-programming commented:
@love-sonkar The second image you shared is not a problem so ignore that

The sort and language buttons look fine but the header feels too cluttered

love-sonkar commented:
we can do one thing, we can remove the contributers button from header when we go 768px

love-sonkar commented:
i have given hidden property to the contributers button and added text wrap to sign in text.

![image](https://github.com/user-attachments/assets/1443e80c-5169-4c51-8278-540118c67a82)

max-programming commented:
@love-sonkar How about having a hamburger menu instead for mobile?

We can use drawer component by DaisyUI
https://daisyui.com/components/drawer/

love-sonkar commented:
Hey i would like to suggest somthing. so our website is all about finding repository for hacktoberfest and i think most of the users visit this site in laptop desktop, so i don't think it's good idea to add an drawer component because our our visitors are all from laptop and desktop.

max-programming commented:
@love-sonkar Even though most visitors use laptop/desktop, there is still a large number of users on phones, and it will increase. 
Not forcing you but if you're not comfortable then maybe create a new issue so someone else will take this up

![image](https://github.com/user-attachments/assets/201ca08a-60b6-4ddf-a94c-1610c2cb5522)

love-sonkar commented:
okay i will create new Issue because i'm little bit busy so i can't implement this

love-sonkar commented:
can you merge and close this PR

max-programming commented:
As of now something on the homepage looks messed up
![image](https://github.com/user-attachments/assets/eced349a-1800-4aab-bc8c-37fd860a6496)

max-programming commented:
@love-sonkar Any updates?
